backend/README.md

```diff
@@ -297,6 +298,31 @@ Run with (supervisord.conf should be provided, see [supervisord.conf default_pat
 
     supervisord
 
+
+Running Pushpin Service
+---------------
+
+For a working Tracim instance, you need to setup pushpin as proxy for tracim web service.
+
+### Development purpose
+for dev, it's recommended to follow [this documentation](./doc/live_message_setup.md) for docker installation of pushpin
+
+### Production install
+
+for production, you can install pushpin daemon directly.
+
+You should install pushpin, for example in debian/ubuntu:
+
+    apt install pushpin
+
+and configure it to proxy tracim web (by default tracim web run on port 6543):
+
+    echo "* localhost:6543" > /etc/pushpin/routes
+
+and after this change you need to restart your pushpin service:
+
+    systemctl restart pushpin
+
```

Comment:
This is duplicated from the main README, is it necessary here ?

Reply:
Normally this should be more detailled here as this is the specific backend documentation. I'm not sure how to organize to make both backend doc and defaut readme usable.